rasa/model_training.py

```diff
@@ -590,7 +590,9 @@ async def _train_core_with_validated_data(
                 await rasa.core.train.train_in_chunks(
                     domain_file=domain,
                     training_resource=file_importer,
-                    output_path=Path(_train_path, DEFAULT_CORE_SUBDIRECTORY_NAME),
+                    output_path=os.path.join(
```

Comment:
Can you leave it to `Path` to be consistent with the code? Any particular reason why it should change?

Reply:
this is the change from `main`